Apps/Src/DebugDump.c

```diff
@@ -0,0 +1,130 @@
+#include "os.h"
+#include "Tasks.h"
+#include "bsp.h"
+#include "CANbus.h"
+#include "CAN_Queue.h"
+#include "Pedals.h"
+#include "Minions.h"
+#include "Contactors.h"
+#include "common.h"
+#include <errno.h> 
+#include "Tasks.h"
+#include "SendTritium.h"
+#include "common.h"
+
+// global variables
+extern fault_bitmap_t FaultBitmap;
+extern os_error_loc_t OSErrLocBitmap;
+
+static const char *MINIONPIN_STRING[] = {
+    FOREACH_MinionPin(GENERATE_STRING)
+};
+
+static const char *CONTACTOR_STRING[] = {
+    FOREACH_contactor(GENERATE_STRING)
+};
+
+static const char *GEAR_STRING[] = {
+    FOREACH_Gear(GENERATE_STRING)
+};
```

Comment:
I understand the utility for the Minion Pin one, but the other two are so small that I'm not sure it's really all that useful to have it macro-ed away. Maybe if there was a more generic foreach macro that we could pass an enum definition into alongside the operator function?

Reply:
I agree that it is sort of tedious to do it on these enums since they are so small, but it is best to just use it for the sake of scalability. I'll look into a better way to make it cleaner. It currently seems difficult because you'd have to pass in a variable number of variables into a macro. I do not currently see a way to access individual variables within the macro (just can pass the variables as a whole to another function).